app/controllers/concerns/qa_states.rb

```diff
@@ -0,0 +1,28 @@
+module QAStates
```

Comment:
Any reason for `QAStates` over `QA::States`? Or what about a more rails concern-y name like `Publishable` or